packages/arb-token-bridge-ui/package.json

```diff
@@ -64,10 +64,10 @@
     "generateDenylist": "ts-node --project ./scripts/tsconfig.json ./scripts/generateDenylist.ts",
     "generateCoreChainsToMonitor": "ts-node --project ./scripts/tsconfig.json ./scripts/generateCoreChainsToMonitor.ts",
     "generateOrbitChainsToMonitor": "ts-node --project ./scripts/tsconfig.json ./scripts/generateOrbitChainsToMonitor.ts",
-    "e2e:open": "env-cmd -f .e2e.env yarn synpress open --configFile synpress.config.ts",
-    "e2e:run": "env-cmd -f .e2e.env yarn synpress run --configFile synpress.config.ts",
-    "e2e:run:cctp": "E2E_CCTP=true yarn e2e:run",
-    "e2e:install": "cypress install"
+    "e2e:env": "if [ \"$CI\" != \"true\" ]; then env-cmd -f .e2e.env; fi",
+    "e2e:open": "yarn e2e:env && yarn synpress open --configFile synpress.config.ts",
+    "e2e:run": "yarn e2e:env && yarn synpress run --configFile synpress.config.ts",
+    "e2e:run:cctp": "E2E_CCTP=true yarn e2e:run"
```

Comment:
made these a bit simpler. basically, if you're running with `CI=true` then you pass in all env variables, otherwise (locally) it loads from .e2e.env

this makes it simpler, we run the exact same command